Source/GNUmakefile

```diff
@@ -284,6 +286,9 @@ GSDisplayServer.m \
 GSHelpManagerPanel.m \
 GSInfoPanel.m \
 GSMemoryPanel.m \
+GSStoryboardDocument.m \
```

Comment:
These classes are not part of the branch at the moment.

Reply:
Removed them from my local and from the GNUmakefile they aren't needed.  I was considering another approach, but the current way seems better.   Storyboards are simply packed XIB files, so I am transforming them into proper XIBs before loading.